<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>761</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string>ME909S-821 check</string>
    </property>
    <widget class="QCheckBox" name="checkBox_deviceNode">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>deviceNode</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_ltemodule">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>LTEmodule</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_SIMSlot">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SIMSlot</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_SIMDataService">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SIMDataService</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_SIMOperator">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SIMOperator</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_SIMDialing">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SIMDialing</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_SIMSignal">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SIMSignal</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_netAccess">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>NetAccess</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_deviceNode">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_LTEmodule">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_SIMslot">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_service">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_operator">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_dialing">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_signal">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_netaccess">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_temp">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_temp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>temperature</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_iccid">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ICCID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_iccid">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_currentMode">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>540</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_currentMode">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>540</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>currentMode</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_simSwitch">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SIMSwitch</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_simSwitch">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_pingResult">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>510</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>pingResult</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pingResult">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>510</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_cclk">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CCLK</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cclk">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>360</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_eons">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>390</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>EONS</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_eons">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>390</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_cardmode">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>420</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CARDMODE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cardmode">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>420</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_simst">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>450</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>SIMST</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_simst">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>450</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_cme_error">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>480</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CMEERROR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cme_error">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>480</y>
       <width>601</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_version">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>51</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Version</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>160</y>
      <width>441</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string>Notes:</string>
    </property>
    <widget class="QTextEdit" name="textEdit_info">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>421</width>
       <height>441</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sysTime">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>101</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>sysTime</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>50</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Counter:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_successCnt">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>successCnt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_noteSuccess">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>OK:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_noteFailed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Failed:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_failedCnt">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>failedCnt</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1257</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
